lib/matplotlib/mathtext.py

```diff
@@ -887,12 +887,12 @@ class DejaVuFonts(UnicodeFonts):
 
     def __init__(self, *args, **kwargs):
         # This must come first so the backend's owner is set correctly
+        TruetypeFonts.__init__(self, *args, **kwargs)
```

Comment:
Actually, I think it turned out that this move wasn't necessary after all -- I had meant to revert it before committing.  I'll do so now.